bundles/org.openhab.binding.sagercaster/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -142,7 +143,7 @@
 	<channel-type id="wind-direction">
 		<item-type>String</item-type>
 		<label>@text/windDirectionLabel</label>
-		<category>Wind</category>
+		<category>wind</category>
```

Comment:
Please keep camelcase as it is like that everywhere in all bindings.

Reply:
Done.